package.json

```diff
@@ -25,12 +25,12 @@
   "scripts": {
     "typecheck": "tsc -p tsconfig.esm.json --noEmit",
     "bootstrap": "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true yarn && ocular-bootstrap",
-    "build": "ocular-clean && tsc -b tsconfig.esm.json && tsc -b tsconfig.es5.json",
+    "build": "tsc -b tsconfig.esm.json && tsc -b tsconfig.es5.json",
```

Comment:
Removed some ocular commands since we don't really know how to use them and they were getting in the way.